Processor/processor.py

```diff
@@ -29,7 +56,12 @@ def __process_image_tag(self, tag):
             if tag.has_attr('ec-data-src'):
                 src = tag['ec-data-src']
         if src:
-            self.body += Constants.IMG_JSON_LD_TEMPLATE \
+            if CommonUtility.is_url(src):
+                img, ext = CommonUtility.read_image(src)
+                if self.aspect_ration_check(img):
```

Comment:
Else condition missed, where image isn't above aspect ration, and yet it will not be part of json-ld.